### ISSUE TITLE
Project abandoned - But no license included

### ISSUE THREAD

**Issue by pykong** [bobef/ColorSchemeEditor#13]
_Several attempts to reach @bobef have failed. Issues and pull requests remain open without any reaction from the author for more than two years._

In addition, I wrote two emails to the email address stated in the author's profile, in which I asked him to come back. I even offered to take over the project to serve as the new maintainer in case he no longer wishes to fulfill this role. I also tried to reach him via a private message on Sublime Forum.

Yet, those requests have not elicited any response by the author.

This is very sad, as ColorSchemeEditor is a particularly helpful tool, for all Sublime users who love to edit color schemes. 

**Unfortunately, the repository does not feature any license.**
**This forbids, in my humble understanding, to simply publish a rebase to PackageControl.io.**
See: [https://choosealicense.com/no-license/](https://choosealicense.com/no-license/)
Also See: [https://opensource.guide/legal/#are-public-github-projects-open-source](https://opensource.guide/legal/#are-public-github-projects-open-source)

**TL:DR:** We may be stuck for now with the status quo and an unresponsive author.

**What To Do?**
Those who like to enjoy a functioning version of ColorSchemEditor, which has some major bugs fixed, may be referred to my open pull request and apply file changes manually: 
[https://github.com/bobef/ColorSchemeEditor/pull/12/files](https://github.com/bobef/ColorSchemeEditor/pull/12/files)

Or use an alternative Plug-In like [SchemeEditor](https://github.com/facelessuser/SchemeEditor).

**Comment by pykong** [comment]
Unfortunately, as of this day, there is still no word from @bobef.
